### PR TITLE
Vanshika patch

### PR DESCRIPTION
**Pull request by vansjyo**
## issues resolved

### CODE REVIEW

File: signin.html [hunk 1]
```diff
@@ -1,180 +1,102 @@
...
+				<img src="images\collage.jpg" height="350px" width="100%">
+			</div><br>
+		</div>
+
+		<br><br><br>
+	</main>
+	
+
+	<!-- footer -->
+	<script language="javascript" type="text/javascript" src="txt\footer.txt"></script>
```
Comment (salman-bhai) [inline signin.html]: Shouldn't this be `footer.html`?

File: signup.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+    <title>NITK COOPERATIVE SOCIETY | ONLINE DELIVERY</title>
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width" initial-scale="1.0">
+    <meta name="description" content="online coop service nitk">
+    <meta name="keywords" content="nitk online coop">
+
+
+    <link rel="icon" href="C:\Users\laptop\Desktop\website\favicon.ico" type="image/x-icon" />
```
Comment (salman-bhai) [inline signup.html]: Change to relative link!

File: signup.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+
+
+   </head>
+
+   <body id="back">
+
+     <!-- header  -->
+     <script language="javascript" type="text/javascript" src="txt\header.txt"></script>
```
Comment (salman-bhai) [inline signup.html]: Change to `header.html`

File: signup.html [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+             <div class="col s12 m5"><img src="images\buysell.jpg" width="100%">
+             </div>
+           </div>
+         </div><br><br><br>
+
+       </main>
+
+
+       <!-- footer -->
+       <script language="javascript" type="text/javascript" src="txt\footer.txt"></script>
```
Comment (salman-bhai) [inline signup.html]: Change to `footer.html`

File: index.html [hunk 5]
```diff
@@ -2,269 +2,187 @@
...
-
-
-<script type="text/javascript" src="https://code.jquery.com/jquery-2.1.1.min.js"></script>           
-<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.3/js/materialize.min.js"></script> 
-<script src="js\jquery.flexslider.js"></script>
-<script src="js\main.js"></script>
-<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.4/angular.min.js"></script>
-
+	<!-- header  -->
+	<script language="javascript" type="text/javascript" src="txt\header.txt"></script>
```
[the remark this answers is not in the document]
Comment (vansjyo) [reply index.html]: no

**Pull request merged**